src/main/java/com/dataloom/datastore/search/controllers/SearchController.java

```diff
@@ -104,4 +104,18 @@ public String executeQueryJson( @RequestBody SearchRequest request ) {
     @Override public AuthorizationManager getAuthorizationManager() {
         return authorizations;
     }
+
+    @RequestMapping(
+            path = { ORGANIZATIONS },
+            method = RequestMethod.POST,
+            produces = { MediaType.APPLICATION_JSON_VALUE } )
+    @Override
+    public String executeOrganizationSearch( @RequestBody String searchTerm ) {
+        try {
+            return ObjectMappers.getJsonMapper().writeValueAsString( searchService.executeOrganizationKeywordSearch( searchTerm ) );
```

Comment:
So it makes sense to return a string here, but you should be passing through the string returned from elasticsearch. No point in deserializing unless we are going to introspect on contents.